edgar_tool/constants.py

```diff
@@ -1608,3 +1606,18 @@
     ],
     "Trust indenture filings": ["305B2", "T-3"],
 }
+
+# These are the verbose descriptions of the categories that are used in the CLI,
+# not used in the CLI but included for completeness
+TEXT_SEARCH_FILING_VS_MAPPING_CATEGORIES_MAPPING = {
+    "all_annual_quarterly_and_current_reports": list(TEXT_SEARCH_CATEGORY_FORM_GROUPINGS.keys())[0],
+    "all_section_16": list(TEXT_SEARCH_CATEGORY_FORM_GROUPINGS.keys())[1],
+    "beneficial_ownership_reports": list(TEXT_SEARCH_CATEGORY_FORM_GROUPINGS.keys())[2],
+    "exempt_offerings": list(TEXT_SEARCH_CATEGORY_FORM_GROUPINGS.keys())[3],
+    "registration_statements": list(TEXT_SEARCH_CATEGORY_FORM_GROUPINGS.keys())[4],
+    "filing_review_correspondence": list(TEXT_SEARCH_CATEGORY_FORM_GROUPINGS.keys())[5],
+    "sec_orders_and_notices": list(TEXT_SEARCH_CATEGORY_FORM_GROUPINGS.keys())[6],
+    "proxy_materials": list(TEXT_SEARCH_CATEGORY_FORM_GROUPINGS.keys())[7],
+    "tender_offers_and_going_private_tx": list(TEXT_SEARCH_CATEGORY_FORM_GROUPINGS.keys())[8],
+    "trust_indentures": list(TEXT_SEARCH_CATEGORY_FORM_GROUPINGS.keys())[9],
+}
```

Comment:
improved, it was previously hard-coded

Reply:
I actually think it is better to be hard-coded as it is a mapping.

For example, this approach could cause problems if the order of items in `TEXT_SEARCH_CATEGORY_FORM_GROUPINGS` changed for some reason.

If you could revert, that would be great.